### PR TITLE
add service configs

### PR DESCRIPTION
**Pull request by syamgk**

### CODE REVIEW

File: chat-services/mattermost-gitlab.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+services:
+- hash: f2cf7f90f65214143db61c42fb06d0dda3eebaa6
+  hash_length: 7
+  name:
```
Comment (vpavlin) [inline chat-services/mattermost-gitlab.yaml]: Please add names - for `chat-integrations` repository, names should be in a form of `reponame-subdir` - e.g. `chat-integrations-github`

Comment (syamgk) [reply chat-services/mattermost-gitlab.yaml]: okey

File: chat-services/mattermost.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+services:
+- hash: a221624741d96068653cde7bc47061fcc0ea59dc
+  hash_length: 7
+  name: mattermost
```
Comment (vpavlin) [inline chat-services/mattermost.yaml]: This should be `chat`

File: chat-services/mattermost-gitlab.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+services:
+- hash: f2cf7f90f65214143db61c42fb06d0dda3eebaa6
+  hash_length: 7
+  name: chat-integrations-gitlab
```
Comment (vpavlin) [inline chat-services/mattermost-gitlab.yaml:4]: this should be `chat-integrations-github`

File: chat-services/mattermost-github.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+services:
+- hash: f2cf7f90f65214143db61c42fb06d0dda3eebaa6
+  hash_length: 7
+  name: chat-integrations-github
```
Comment (vpavlin) [inline chat-services/mattermost-github.yaml:4]: this should be `chat-integrations-gitlab`

File: chat-services/mattermost.yaml (hunk 2 above)
Comment (syamgk) [reply chat-services/mattermost.yaml]: fixed

File: chat-services/mattermost-gitlab.yaml (hunk 3 above)
Comment (syamgk) [reply chat-services/mattermost-gitlab.yaml:4]: fixed the typo

File: chat-services/mattermost-github.yaml (hunk 4 above)
Comment (syamgk) [reply chat-services/mattermost-github.yaml:4]: fixed the typo

**Pull request closed without merging**